src/core_write.cpp

```diff
@@ -162,6 +172,7 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
     entry.pushKV("size", (int)::GetSerializeSize(tx, SER_NETWORK, PROTOCOL_VERSION));
     entry.pushKV("vsize", (GetTransactionWeight(tx) + WITNESS_SCALE_FACTOR - 1) / WITNESS_SCALE_FACTOR);
     entry.pushKV("locktime", (int64_t)tx.nLockTime);
+    entry.pushKV("replaceable", SignalsOptInRBF(tx));
```

Comment:
Imo this should be the same key-value as the `bip125-replaceable` in the rpcwallet

Reply:
Good idea! I updated the PR.